analyses/molecular-subtyping-chordoma/01-Subtype-chordoma.Rmd

```diff
@@ -0,0 +1,231 @@
+rm(expression_data)
+```
+
+The `smarcb1_expression` is a not a friendly form ^^; Transposing needed: 
+
+```{r}
+# transpose such that samples are rows
+smarcb1_expression <- t(smarcb1_expression) %>%
+  # make a data.frame
+  as.data.frame() %>%
+  # we want the rownames that are biospecimen identifers as their own column called Kids_First_Biospecimen_ID
+  tibble::rownames_to_column("Kids_First_Biospecimen_ID") %>%
+  # give SMARCB1 column a slightly better column name
+  rename(SMARCB1_expression = SMARCB1)
+smarcb1_expression
+```
+This also needs sample_id to add it in:
+
+```{r}
+smarcb1_expression <- smarcb1_expression %>%
+  inner_join(select(chordoma_id_df,
+                    Kids_First_Biospecimen_ID,
+                    sample_id),
+             by = "Kids_First_Biospecimen_ID")
+smarcb1_expression
+```
+
+```{r}
+chordoma_smarcb1_df <- smarcb1_expression %>%
+  # any missing samples will get filled with NA when using a full join
+  full_join(chordoma_copy, by = "sample_id")
+chordoma_smarcb1_df
+```
+
+```{r}
+# this step adds in the participant identifier (sample_id to match between the two data.frame)
+chordoma_smarcb1_df <- chordoma_smarcb1_df %>%
+  inner_join(distinct(select(chordoma_id_df, 
+                             sample_id, 
+                             Kids_First_Participant_ID)),
+             by = "sample_id")
+
+# combining the two biospecimen identifiers to a single column (all biospecimen IDs for a sampl separated by a comma)
+chordoma_smarcb1_df <- chordoma_smarcb1_df %>%
+  mutate(Kids_First_Biospecimen_ID = if_else(
+    # one sample is missing WGS data, so if that's true only include biospecimen ID from RNA-seq
+    is.na(biospecimen_id),
+    Kids_First_Biospecimen_ID,
+    paste(Kids_First_Biospecimen_ID, biospecimen_id, sep = ", ")
+  ))
+chordoma_smarcb1_df
+```
+
+```{r}
+chordoma_smarcb1_df <- chordoma_smarcb1_df %>%
+  select(Kids_First_Participant_ID, 
+         Kids_First_Biospecimen_ID, 
+         sample_id,
+         status,
+         SMARCB1_expression) %>%
+  # 'status' is replaced a more descriptive name
+  rename(focal_SMARCB1_status = status)
+chordoma_smarcb1_df
+```
+
+### Plot _SMARCB1_ expression
+
+copy loss vs. copy neutral 
+
+```{r}
+# this specifies that this is the data we want to plot
+chordoma_smarcb1_df %>%
+  # drop the sample that doesn't have WGS data
+  tidyr::drop_na() %>%
+  # this step specifies what should go on the x- and y-axes
+  ggplot(aes(x = focal_SMARCB1_status,
+             y = SMARCB1_expression)) +
+  # we want a jitter plot where the points aren't too far 
+  # apart that's what width does
+  geom_jitter(width = 0.1) +
+  # this is plotting the median as a blue diamond
+  stat_summary(fun.y = "median", 
+               geom = "point", 
+               size = 3, 
+               color = "blue", 
+               shape = 18)
+```
+
+```{r}
+ggsave(filename = plot_file)
+```
+
+Write the table to file.
+
+```{r}
```

Comment:
Should this output file include a subtype "call"? Or is the deletion status sufficient?

Reply:
What do you mean by subtype "call"?

Reply:
I think @jashapiro is asking about whether we need a column that indicates whether or not a tumor is poorly-differentiated. I was hesitant to add this until the copy number was in better shape. We now have copy number consensus files (contain calls that multiple methods agree on) that we could use instead of the old version that uses the URL.

Reply:
I updated the notebook to use the consensus file and also updated the documentation to reflect that change in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/475/commits/c26db8553ac19c4c0e39234693a0308d2f34c98f

Reply:
There are some instances where the expression levels between losses and the neutral calls are similar (current state of plot below): 
![smarcb1_expression_copy_status](https://user-images.githubusercontent.com/19534205/73467439-5246b900-4351-11ea-9c2c-1666133d6f3a.png)

We may want to hold off on adding calls until we do a bit more digging as part of #387 and #486

Reply:
Agreed